phpunit.xml.dist

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <phpunit colors="true" bootstrap="vendor/autoload.php">
+    <php>
+        <!-- Disable E_USER_DEPRECATED until reaching compatibility with Symfony 3.0 -->
+        <!-- php -r 'echo -1 & ~E_USER_DEPRECATED;' -->
+        <ini name="error_reporting" value="-16385"/>
```

Comment:
Please let's not do this, really, as we will also be unable to find deprecations in other software.
Instead, lock onto <2.7 until 2.7 is out.